python/tests/plugins/lsp_server/providers/test_semantic_tokens_provider.py

```diff
@@ -61,7 +61,7 @@ async def test_get_semantic_tokens_for_enum(self):
 
         props = {
             enum.name: {
-                enum.get_root_key(): (SemanticTokenTypes.Keyword, 0b11),
+                enum.get_root_key(): (SemanticTokenTypes.Keyword, 0b10),
```

Comment:
Updated this to reflect the change in behavior in the token provider. I believe the 3 -> 2 change is because a number of these names no longer match the root key, which means it matches with one fewer of the bitmask checks.

Reply:
I think we still want this to be `0b11` as the bitmask indicates that a token is [readonly](https://github.com/jondavid-black/AaC/blob/e743b99/python/src/aac/plugins/first_party/lsp_server/providers/semantic_tokens_provider.py#L51-L58) and it is [defined in the default library](https://github.com/jondavid-black/AaC/blob/e743b99/python/src/aac/plugins/first_party/lsp_server/providers/semantic_tokens_provider.py#L61-L65) - i.e. defined in the core spec. As such, I think these root keys should always have both modifiers - i.e. `0b11`. I think we need a slight modification of the logic in those two functions (linked above).

Reply:
Hmm. Why are we considering root keys as readonly?